hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/fs/viewfs/TestViewFsLinkFallback.java

```diff
@@ -182,6 +184,26 @@ public void testMkdirOfNewDirWithOutMatchingToMountOrFallbackDirTree()
     assertTrue(fsTarget.exists(test));
   }
 
+  /**
+   * Test getDelegationToken when fallback is configured
+   */
+  @Test
+  public void testGetDelegationToken() throws IOException {
```

Comment:
If we are specifically testing for fallback, can you please change the name to include fallback?